src/azul/service/responseobjects/hca_response_v5.py

```diff
@@ -478,10 +628,17 @@ def map_entries(self, entry):
             bundles=self.make_bundles(entry)
         )
 
+    @staticmethod
+    def for_entity_type(hits, entity_type):
+        if entity_type == 'projects':
+            return ProjectKeywordSearchResponse(hits)
+        return ProjectKeywordSearchResponse(hits)
```

Comment:
Why does this return ProjectKeywordSearchResponse in both cases? If that's a mistake, why don't the tests catch it?